src/impl_error_chain_kind.rs

```diff
@@ -103,7 +102,7 @@ macro_rules! impl_error_chain_kind {
     ) => {
         impl_error_chain_kind!(SORT [$( $def )*]
             items [$( $(#[$imeta])* => $iitem: $imode [$( $ivar:$ityp ),*] {$( $ifuncs )*} )*]
-            buf [$( #[$bmeta] )* => $bitem: TUPLE [$( $qvar:$qtyp ),*] ]
```

Comment:
All following errors are similar to this one, so I'm just providing the output for this error.
```
error: meta-variable repeats with different Kleene operator
   --> src/impl_error_chain_kind.rs:106:55
    |
102 |         queue [($( $qvar:ident: $qtyp:ty ),+) $( $tail:tt )*]
    |                                            - expected repetition
...
106 |             buf [$( #[$bmeta] )* => $bitem: TUPLE [$( $qvar:$qtyp ),*] ]
    |                                                       ^^^^^         - conflicting repetition
```